### PR TITLE
Add "Contributing to Documentation" page

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Here is "Contributing to Documentation" page for [#1354](https://github.com/apache/zeppelin/pull/1354) 👯

### CODE REVIEW

File: contribution/documentation.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+layout: sideMenu
+title: "Documentation"
+description: ""
+group: nav-contrib
+menu: nav-contrib-front
```
Comment (corneadoug) [inline contribution/documentation.md]: You might want to remove this property otherwise you will have a side menu with front-end Guidelines content :)

Comment (AhyoungRyu) [reply contribution/documentation.md]: Yeah right. I did it in the last commit 💃

**Comment by corneadoug** [comment]
@AhyoungRyu Ready to Merge?

**Comment by AhyoungRyu** [comment]
Yup please do owner :D

**Pull request merged**